src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -61,10 +62,10 @@ class FunSpec private constructor(builder: Builder) {
     codeWriter: CodeWriter,
     enclosingName: String?,
     implicitModifiers: Set<KModifier>,
-    includeParametersKdoc: Boolean
+    includeExtrasKdoc: Boolean = false
```

Comment:
Not sold on "extras", any other ideas regarding the naming?

Reply:
These are called "tags" in Javadoc, so I suggest `includeKdocTags`. The function below should be renamed from `kdocWithParameters()` to `kdocWithTags()`.